### PR TITLE
[Tests-Only] Refactor webUI scenarios to prevent unnecessary skeleton file creation

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

Refactor the following suite tests to not use skeleton files:

- webUILogin
- webUIMarkdownEditor
- webUIMoveFilesFolders
- webUINotifications
## Related Issue

- Part of https://github.com/owncloud/QA/issues/659

## How Has This Been Tested?

- CI
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14794/

<p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14801/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14804/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14804/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14804/

<p>






</p></details>

File: tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature [hunk 1]
```diff
@@ -75,49 +83,53 @@ Feature: move files
...
+    Given user "Alice" has uploaded file "data.zip" to "simple-folder/data.zip"
+    And user "Alice" has created folder "simple-folder/simple-empty-folder"
+    And user "Alice" has shared folder "simple-folder" with link with "read, update, create, delete" permissions
     And the public uses the webUI to access the last public link created by user "Alice"
     And the user moves file "data.zip" into folder "simple-empty-folder" using the webUI
     Then breadcrumb for folder "simple-empty-folder" should be displayed on the webUI
     And file "data.zip" should be listed on the webUI
-    And as "Alice" file "simple-folder/simple-empty-folder/data.zip" should exist
-    But as "Alice" file "simple-folder/data.zip" should not exist
+    And as "Alice" file "simple-folder/data.zip" should not exist
```
Comment (swoichha) [inline tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature:93]:
```suggestion
   And as "Alice" file "simple-folder/simple-empty-folder/data.zip" should exist
   But as "Alice" file "simple-folder/data.zip" should not exist
```

File: tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature [hunk 2]
```diff
@@ -127,14 +139,15 @@ Feature: move files
   Scenario: sharee moves a file shared by sharer into another folder
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
     And the administrator has set the default folder for received shares to "Shares"
-    And user "Brian" has been created with default attributes
-    And user "Alice" has uploaded file with content "test content" to "simple-empty-folder/testFile.txt"
-    And user "Alice" has shared folder "/simple-empty-folder" with user "Brian"
-    And user "Brian" has accepted the share "simple-empty-folder" offered by user "Alice"
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created file "testFile.txt"
```
Comment (swoichha) [inline tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature:143]: Why create this new file?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14894/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14896/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14924/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14924/

<p>







</p></details>

**Review by Talank — COMMENTED** [review]
LGTM. Make CI happy.

**Review by saw-jan — COMMENTED** [review]
LGTM :+1:

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14954/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15001/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15001/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15013/

<p>










</p></details>

**Review by individual-it — APPROVED** [review]

**Pull request merged**